data/resources/ui/content-chat-action-bar.ui

```diff
@@ -33,5 +33,14 @@
         </style>
       </object>
     </child>
+    <child>
+      <object class="GtkLabel" id="restriction_label">
```

Comment:
I suggest adding `<property name="justify">center</property>`, this way longer labels look a bit better.

Reply:
I also found `<property name="visible">False</property>` to be useless since we have binding for that property